odoo/addons/base/ir/ir_model.py

```diff
@@ -912,10 +912,14 @@ def _add_manual_fields(self, model):
         fields_data = self._get_manual_field_data(model._name)
         for name, field_data in fields_data.items():
             if name not in model._fields and field_data['state'] == 'manual':
-                field = self._instanciate(field_data)
-                if field:
-                    model._add_field(name, field)
-
+                try:
+                    field = self._instanciate(field_data)
+                    if field:
+                        model._add_field(name, field)
+                except Exception:
+                    type(self)._bad_fields.add(name)
+                    _logger.warning("Manual field '{}' : loading skipped".format(
```

Comment:
`_logger` has its own formatting:

```
_logger.warning("Manual field '%s' : loading skipped", field_data['name'])
```
In addition, the Odoo guidelines tells to prefer the use `%` or `%(myvariable)` rather than `format()`
e.g.
- `"Manual field '%s' : loading skipped" % field_data['name']`
- `"Manual field '%(field_name)s' : loading skipped" % {'field_name': field_data['name']}`

https://www.odoo.com/documentation/13.0/reference/guidelines.html#use-translation-method-correctly
```quote
In general in Odoo, when manipulating strings, prefer % over .format() 
(when only one variable to replace in a string),
and prefer %(varname) instead of position (when multiple variables have to be replaced). 
```
One of the reason is that `format` is not as safe:
- http://lucumr.pocoo.org/2016/12/29/careful-with-str-format/
- https://python-forum.io/Thread-str-format-security-vulnerability